cpp/CMakeLists.txt

```diff
@@ -444,12 +432,9 @@ cuGraph library is a collection of GPU accelerated graph algorithms that process
 
 ]=])
 
-set(code_string
-[=[
-thrust_create_target(cugraph::Thrust FROM_OPTIONS)
-]=])
+set(code_string "")
```

Comment:
is it better to do this, or just remove it altogether along with the `FINAL_CODE_BLOCK` args to the `rapids_export` calls below?

Reply:
Removed!